<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabVisualizzaDatiPersonali</class>
 <widget class="QTabWidget" name="tabVisualizzaDatiPersonali">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizza dati personali</string>
  </property>
  <widget class="QWidget" name="VisualizzaDatiPersonaliCamerieri">
   <attribute name="title">
    <string>Visualizza dati personali</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnTornareIndietro">
      <property name="text">
       <string>Tornare Indietro</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupboxSchedaVisualizzazioneDatiPersonaliCamerieri">
      <property name="title">
       <string>Scheda dati personali cameriere:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="0">
        <widget class="QLabel" name="labelRuoloCamerieri">
         <property name="text">
          <string>Ruolo:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEditNomeCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelNomeCamerieri">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelIBANCamerieri">
         <property name="text">
          <string>IBAN:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditCognomeCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Cognome</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelDataDiNascitaProvinciaCamerieri">
         <property name="text">
          <string>Data di nascita:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelCognomeCamerieri">
         <property name="text">
          <string>Cognome:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelLuogoDiNascitaCamerieri">
         <property name="text">
          <string>Luogo di nascita(provincia):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelIndirizzoEmailCamerieri">
         <property name="text">
          <string>Indirizzo Email:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelTelefonoCamerieri">
         <property name="text">
          <string>Telefono:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditLuogoDiNascitaProvinciaCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Luogo di nascita(Provincia):</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelTurnoDiLavoroCamerieri">
         <property name="text">
          <string>Turno di lavoro:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEditTelefonoCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Telefono</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEditIndirizzoEmailCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>myEmail@domain.com</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEditIBANCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>IBAN</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDateEdit" name="dateEditDataDiNascitaCamerieri">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="lineEditRuoloCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Ruolo</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="lineEditTurnoDiLavoroCamerieri">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Mattina / Pomerggio</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
